installing/installing_azure/installing-azure-network-customizations.adoc

```diff
@@ -46,7 +46,7 @@ include::modules/configuring-hybrid-ovnkubernetes.adoc[leveloffset=+1]
 
 [NOTE]
 ====
-For more information on using Linux and Windows nodes in the same cluster, see xref:../../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Understanding Windows container workloads].
+For more information on using Linux and Windows nodes in the same cluster, see Understanding Windows container workloads.
```

Comment:
Should you remove this note as the referenced section won't be available for searching? (I assume)